test_requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Test-only dependencies should go here, not in setup.py
-pytest>=3.2.2,<4.0.0
```

Comment:
Should we increase the min version too?

Reply:
4.0.0 was out last year, seems like a good min version

Reply:
Sure thing, will fix